docs/platforms/node/common/configuration/filtering.mdx

```diff
@@ -8,15 +8,21 @@ When you add Sentry to your app, you get a lot of valuable information about err
 
 The Sentry SDKs have several configuration options to help you filter out events.
 
-We also offer [Inbound Filters](/product/data-management-settings/filtering/) to filter events in sentry.io. We recommend filtering at the client level though, because it removes the overhead of sending events you don't actually want. Learn more about the [fields available in an event](https://develop.sentry.dev/sdk/event-payloads/).
+We also offer [Inbound Filters](/product/data-management-settings/filtering/) to filter events in sentry.io. There are both some predefined inbound filters (e.g. filtering known browser extensions), as well as adding your own message-based filters. We recommend filtering at the client level though, because it removes the overhead of sending events you don't actually want. Learn more about the [fields available in an event](https://develop.sentry.dev/sdk/event-payloads/).
```

Comment:
```suggestion
Sentry offers [Inbound Filters](/product/data-management-settings/filtering/) that you can enable per project to filter out various events in sentry.io. You can use our pre-defined inbound filters (e.g. filtering known browser extensions), as well as add your own message-based filters. 

However, we recommend filtering at the client-level, because it removes the overhead of sending events you don't actually want. The Sentry SDKs have several configuration options, which are described in this document, to help you filter out events. To learn more about the event fields you can use for filtering, see [Event Payloads](https://develop.sentry.dev/sdk/event-payloads/).```